toolkit/templates/character_generator.html

```diff
@@ -3,4 +3,651 @@
+    <div class="character-stat">
+      <img src
+           {% static "Character_UI_Stat.png" %}
+           alt="Character UI Stat"
+           style="width: 100%"/>
+      <div class="character-stat-title">STRENGTH</div>
+      <div class="character-stat-base">
+        <h3>10</h3>
+      </div>
+      <div class="character-stat-bonus">
+        <h4>+0</h4>
+      </div>
+    </div>
+    <div class="character-stat">
+      <img src
+           {% static "Character_UI_Stat.png" %}
+           alt="Character UI Stat"
+           style="width: 100%"/>
+      <div class="character-stat-title">DEXTERITY</div>
+      <div class="character-stat-base">
+        <h3>10</h3>
+      </div>
+      <div class="character-stat-bonus">
+        <h4>+0</h4>
+      </div>
+    </div>
+    <div class="character-stat">
+      <img src
+           {% static "Character_UI_Stat.png" %}
+           alt="Character UI Stat"
+           style="width: 100%"/>
+      <div class="character-stat-title">CONSTITUTION</div>
+      <div class="character-stat-base">
+        <h3>10</h3>
+      </div>
+      <div class="character-stat-bonus">
+        <h4>+0</h4>
+      </div>
+    </div>
+    <div class="character-stat">
+      <img src
+           {% static "Character_UI_Stat.png" %}
+           alt="Character UI Stat"
+           style="width: 100%"/>
+      <div class="character-stat-title">INTELLIGENCE</div>
+      <div class="character-stat-base">
+        <h3>10</h3>
+      </div>
+      <div class="character-stat-bonus">
+        <h4>+0</h4>
+      </div>
+    </div>
+    <div class="character-stat">
+      <img src
+           {% static "Character_UI_Stat.png" %}
+           alt="Character UI Stat"
+           style="width: 100%"/>
+      <div class="character-stat-title">WISDOM</div>
+      <div class="character-stat-base">
+        <h3>10</h3>
+      </div>
+      <div class="character-stat-bonus">
+        <h4>+0</h4>
+      </div>
+    </div>
+    <div class="character-stat">
+      <img src
+           {% static "Character_UI_Stat.png" %}
+           alt="Character UI Stat"
+           style="width: 100%"/>
+      <div class="character-stat-title">CHARISMA</div>
+      <div class="character-stat-base">
+        <h3>10</h3>
+      </div>
+      <div class="character-stat-bonus">
+        <h4>+0</h4>
+      </div>
+    </div>
+  </div>
+  <div class="character-baseplate">
+    <div class="character-plate">
+      <div style="text-align: center;">
+        <h4>PROFICIENCY</h4>
+      </div>
+      <div class="character-panel">
+        <div style="width: 25%; float: center;">
+          <h5 style="font-weight: 800;">0</h5>
+        </div>
+        <div style="width: 50%; float: center;">
+          <h5>Proficiency Bonus</h5>
+        </div>
+      </div>
+      <div style="text-align: center;">
+        <h4>SAVING THROWS</h4>
+      </div>
+      <div class="character-panel">
+        <div style="width: 30%; float: left;">
+          <input type="checkbox"
```

Comment:
May be worth looking into bootstrap's check boxes and switches https://getbootstrap.com/docs/5.2/forms/checks-radios/